bot/cogs/cyber.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, bot: Bot):
         with open("assess.json") as f:
             assess_docs = load(f)
 
-        if challenge_num not in range(len(assess_docs) + 1):
+        if challenge_num not in range(len(assess_docs) + 1) and challenge_num != 0:
```

Comment:
I'd prefer it if you included the lower limit in the `range` function:
```suggestion
        if challenge_num not in range(1, len(assess_docs) + 1):
```